tests/integration/helm/upgrade/helm_upgrade_test.go

```diff
@@ -18,16 +18,14 @@
 package helmupgrade
 
 import (
-	"fmt"
-	"strings"
 	"testing"
 
 	"istio.io/istio/pkg/test/framework"
 )
 
 const (
-	previousSupportedVersion = "1.11.3"
-	nMinusTwoVersion         = "1.10.0"
+	previousSupportedVersion = "1.16.0"
+	nMinusTwoVersion         = "1.15.0"
```

Comment:
if Minus Two, we need 1.14? or else change the variable name?

Reply:
It's 2 from the branch. So it'd be 1.15 as master branch points to 1.17.